src/screens/BB-BUS/DetailScreen.js

```diff
@@ -100,6 +100,7 @@ export const DetailScreen = ({ route }) => {
   const rootRouteName = route.params?.rootRouteName ?? '';
   const headerTitle = route.params?.title ?? '';
   const details = route?.params?.data ?? '';
+  const areaId = route.params?.areaId ?? secrets[namespace]?.busBb?.areaId;
```

Comment:
we have `secrets[namespace]?.busBb?.areaId?.toString()` in screens/BB-BUS/IndexScreen.js. should we remove the `toString` there to be in place with here?

Reply:
the api returns `areaIds` as `strings`, so for prop type validation we should treat `areaId` as a string. Instead of removing it there, I added it to the missing places